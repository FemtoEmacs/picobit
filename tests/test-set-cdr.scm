(define x (cons 2 3))
(set-cdr! x 4)
(display (cdr x))
