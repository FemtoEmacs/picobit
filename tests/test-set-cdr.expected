:0080000080
:10800000FBD71D0381194000810F4000810C400007
:10801000810A4000810B4000804200028027011647
:1080200080270117802401188071011A806E011BBE
:108030008027011C8077011D8073011E80240002AF
:10804000806A00028073011F806701208040011058
:1080500080400111803201128027011380660114D3
:108060008069011580780002807D010D8066010E17
:10807000807001098078010902B4015F5202B401E5
:1080800059510607CD504008D140CFB582EED801F6
:1080900020DAB8954121DBB6800221CC9A21CE218D
:1080A00061EE2022CFB67200EF20B6800120DA9E6A
:1080B000A02607E620B558EEA02607E600EF20C070
:1080C000B8BF20B582B648012004C8B8AEA031219F
:1080D000B5A402B583CDDCEF02A034210EC5C122C8
:1080E000CD2102CDB400E87101210EC89220EF210C
:1080F0000E426221B662010421C2EF2002B559B7D7
:108100005520CCB8B8A02C07E620CEB51FEE20CF66
:10811000B6800120D296A02D07E600EF20CC9DA0CE
:108120002407E620CEB505EE20CFB666A106B00046
:108130008FEE20B000ACEEA02D07E600EF20D99521
:10814000A107B1008F20F09C2095A105B1008FA15F
:1081500004B1008FA108B1008F012007E600EF02F3
:108160002004C8212302CDCDB4016C71012204C8C2
:1081700022242302CDCDCDB4017B710123B8DC21B3
:10818000B000F623222602CDCDCDB4018E7101229E
:10819000B8C621B000F621242602CDCDCDB401A170
:1081A00071012022242602CDCDCDCDB401AF7000C7
:1081B0002121C4232502CDCDB401BC710122B89385
:1081C000219220EF2298019420B100F620EF21B8EF
:0E81D0007BB77500B86EB76A21B73B21B725A3
:00000001FF
